docs/UserGuide.md

```diff
@@ -86,12 +85,14 @@ to apply some refactoring.
 
 LSP4IJ provides **experimental**  support for [LSP Semantic Tokens](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_semanticTokens).
 
-### Enable semantic tokens
+### Language Server coloration
+
+LSP4IJ provides default color for semantic tokens that you can configure with `Editor / Color Scheme / Language Server`:
 
-By default, semantic tokens is not activated it.
-To activate it, you need to go to `Editor / Color Scheme / Language Server` and select the `Semantic highlighting` checkbox:
+![Semantic tokens color settings](./images/LSPSemanticTokensColorPage.png)
 
-![Enable LSP semantic tokens](./images/LSPSemanticTokensEnable.png)
+You can see mapping between token type/modifiers and IntelliJ TextAttributeKey (that you can see in the color settings page (ex : Class declaration))

```

Comment:
You can see the mapping between token types/modifiers and IntelliJ's TextAttributeKey on the color settings page (e.g., Class declaration).

Reply:
fixed